src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1796,7 +1796,7 @@ elementFormDefault="qualified">
     </xs:element>
     <xs:element name="VersionPrefix" type="msb:StringPropertyType" substitutionGroup="msb:Property" >
       <xs:annotation>
-        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the end of the verion (i.e. beta)</xs:documentation>
+        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the start of the version (i.e. beta)</xs:documentation>
```

Comment:
Looking at how this is used, I'm not sure this is a good description.

```xml
    <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">1.0.0</VersionPrefix>
    <Version Condition=" '$(VersionSuffix)' != '' ">$(VersionPrefix)-$(VersionSuffix)</Version>
    <Version Condition=" '$(Version)' == '' ">$(VersionPrefix)</Version>
```

So maybe this should be `Initial part of the version number, for example 1.0.0` ?

@nguerrera @eerhardt y'all seem to have been the ones to touch this in the SDK, do you have opinions?